form.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+    <form action="processform.js">
+        <label>First Name</label>
+        <input id="first name" type="text" />
```

Comment:
Cannot have spaces in the "id" field. Update your id's so that there is no space in between. You can use an underscore instead of a space

Reply:
okay sir, correction will be apply.